definitions/region/native_regions/IMACLIM_2.0.yaml

```
@@ -1,13 +1,41 @@
+# The model documentation and regional aggregation for IMACLIM version 2.0
 - IMACLIM 2.0:
-  - IMACLIM 2.0|Africa
-  - IMACLIM 2.0|Brazil
-  - IMACLIM 2.0|Canada
-  - IMACLIM 2.0|China
-  - IMACLIM 2.0|Former Soviet Union excluding EU Baltic countries
-  - IMACLIM 2.0|Europe excluding FSU
-  - IMACLIM 2.0|India
-  - IMACLIM 2.0|Japan, Australia, New Zealand and Korea
-  - IMACLIM 2.0|Middle East
-  - IMACLIM 2.0|Rest of Latin America
-  - IMACLIM 2.0|Rest of Asia
-  - IMACLIM 2.0|USA
+    - IMACLIM 2.0|Africa:
+        countries: [
+            Algeria, Angola, Benin, Botswana, Burkina Faso, Burundi, Cabo Verde, Cameroon, Central African Republic, Chad, Comoros, Côte d'Ivoire, Democratic Republic of the Congo, Djibouti, Egypt, Equatorial Guinea, Eritrea, Eswatini, Ethiopia, Gabon, Gambia, Ghana, Guinea, Guinea-Bissau, Kenya, Lesotho, Liberia, Libya, Madagascar, Malawi, Mali, Mauritania, Mauritius, Morocco, Mozambique, Namibia, Niger, Nigeria, Rwanda, "Saint Helena, Ascension and Tristan da Cunha", Sao Tome and Principe, Senegal, Seychelles, Sierra Leone, Somalia, South Africa, Sudan, Tanzania, Togo, Tunisia, Uganda, Zambia, Zimbabwe, Western Sahara
```

Please use a max-line-width of 88 characters and explicit line breaks for consistency and better readability.
```suggestion
            Algeria, Angola, Benin, Botswana, Burkina Faso, Burundi, Cabo Verde,
            Cameroon, Central African Republic, Chad, Comoros, Côte d'Ivoire,
            Democratic Republic of the Congo, Djibouti, Egypt, Equatorial Guinea,
            Eritrea, Eswatini, Ethiopia, Gabon, Gambia, Ghana, Guinea, Guinea-Bissau,
            Kenya, Lesotho, Liberia, Libya, Madagascar, Malawi, Mali, Mauritania,
            Mauritius, Morocco, Mozambique, Namibia, Niger, Nigeria, Rwanda,
            "Saint Helena, Ascension and Tristan da Cunha", Sao Tome and Principe,
            Senegal, Seychelles, Sierra Leone, Somalia, South Africa, Sudan, Tanzania,
            Togo, Tunisia, Uganda, Zambia, Zimbabwe, Western Sahara
```